### ISSUE TITLE
Singularity: clarify what bind-mounting /home/$USER can mean in practical terms

### ISSUE THREAD

**Issue by willfurnass** [rcgsheffield/sheffield_hpc#1234]
Users might not be aware that when they create Singularity containers on ShARC and Bessemer that their home directories (plus certain other dirs) are bind-mounted.  The sharing of the following the host and containerised environments might cause confusion:

- shell initialisation files e.g. `.bashrc`
- `.Rprofile`
- `.condarc`
- virtualenvs
- `.conda/{envs,pkgs}`
- R libraries
- `~/bin`
- `~/.local/python3.*`

**Comment by willfurnass** [comment]
@jkwmoore suggests that we should also emphasize the importance of looking at startup / environment initialisation scripts within containerised workflows to ensure they don't reference things under the home dir that could be problematic.

**Comment by jkwmoore** [comment]
> @jkwmoore suggests that we should also emphasize the importance of looking at startup / environment initialisation scripts within containerised workflows to ensure they don't reference things under the home dir that could be problematic.

Not sure how we can discuss this particular point concisely to be honest.

**Comment by willfurnass** [comment]
Yeah, agreed.  The other points are perhaps more important to get merged in now.
